block.go

```diff
@@ -397,12 +402,10 @@ func (p *parser) html(out *bytes.Buffer, data []byte, doRender bool) int {
 	return i
 }
 
-// HTML comment, lax form
-func (p *parser) htmlComment(out *bytes.Buffer, data []byte, doRender bool) int {
-	i := p.inlineHtmlComment(out, data)
-	// needs to end with a blank line
-	if j := p.isEmpty(data[i:]); j > 0 {
-		size := i + j
+func (p *parser) renderHtmlBlock(out *bytes.Buffer, data []byte, start int, doRender bool) int {
```

Comment:
Minor style issue, this should be `renderHTMLBlock` since HTML is an initialism. Source: https://github.com/golang/go/wiki/CodeReviewComments#initialisms.